sdk/tests/conformance2/extensions/00_test_list.txt

```diff
@@ -1,4 +1,5 @@
 ext-color-buffer-float.html
+ext-color-buffer-half-float.html
```

Comment:
Could you use --min-version 2.0.1 in this top-of-tree version? This is mainly for bookkeeping. Thanks.